.pylintrc

```diff
@@ -48,7 +48,10 @@ ignore=CVS
 # ignore-list. The regex matches against paths and can be in Posix or Windows
 # format. Because '\' represents the directory delimiter on Windows systems, it
 # can't be used as an escape character.
-ignore-paths=
+# CERTBOT COMMENT
+# Changing this line back to the default of `ignore-paths=` is being tracked by
+# https://github.com/certbot/certbot/issues/7908.
+ignore-paths=.*/_internal/tests/
```

Comment:
At present, this doesn't seem to do anything. If you comment it out, there is no effect. 

I think it is because pylint ignores `acme/acme/_internal` due to that directory missing an `__init__.py` file. Maybe one is required in every directory on the descent down into `tests`?